File: X/clang/XRPL_UKL1/soal3.c
/*
Soal          : 3
Gambar        : A
Matrix kolom  : 6
*/

#include <stdio.h>

int main(){
    int a=5, 
        b=3,
        kolom=6,
        u=a,
        s=b,
        total;
    printf("Deret Aritmatikanya adalah\n");
    for(int i=0;i<kolom;i++){
        for(int j=0;j<i+1;j++){
            printf("%d\t",u);
            u+=b;
            s+=u;
        }
        printf("\n");
    }
    total=s-u; 
    printf("Jumlah Deret Aritmatikanya adalah %d\n", total);
}

File: X/typescript/soal3.ts
/*
Soal          : 3
Gambar        : A
Matrix kolom  : 6
*/

let a = 5,
  b = 3,
  kolom = 6;

let u = a,
  s = b;
console.log("Deret Aritmatikanya adalah");

for (let i = 0; i < kolom; i++) {
  for (let j = 0; j < i + 1; j++) {
    process.stdout.write(u + "\t");
    u += b;
    s += u;
  }
  console.log();
}

s = s - u;
console.log("Jumlah Deret Aritmatikanya adalah " + s);
